db/seeds.rb

```
@@ -6,9 +6,17 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Video.create(title: "Futurama", description: "A show about the future!", small_cover_url: '/public/tmp/futurama.jpg', large_cover_url: '/tmp/futurama.jpg', category: "TV Comedies")
-Video.create(title: "Monk", description: "Not sure what this show is about.", small_cover_url: '/public/tmp/monk.jpg', large_cover_url: '/tmp/monk_large.jpg', category: "TV Dramas")
-Video.create(title: "Family Guy", description: "Family Guy is an American adult animated sitcom created by Seth MacFarlane", small_cover_url: '/tmp/family_guy.jpg', large_cover_url: '/tmp/family_guy.jpg', category: "TV Comedies")
-Video.create(title: "South Park", description: "South Park is an American adult animated sitcom created by Trey Parker and Matt Stone ", small_cover_url: '/tmp/south_park.jpg', large_cover_url: '/tmp/south_park.jpg', category: "TV Comedies")
-Category.create(name: "TV Comedies")
-Category.create(name: "TV Dramas")
\ No newline at end of file
+comedy = Category.create(name: "TV Comedies")
+drama = Category.create(name: "TV Dramas")
+Video.create(title: "Futurama", description: "A show about the future!", small_cover_url: '/tmp/futurama.jpg', large_cover_url: '/tmp/futurama.jpg', category: comedy)
```

I would recommend using bang version of `.create!` here. You get validation errors when you seed if there are any. Now it is silently ignored.

 also styling should be a little different, like this:

```ruby
Video.create!(title: "Futurama",
              description: "A show about the future!",
              small_cover_url: '/tmp/futurama.jpg',
              large_cover_url: '/tmp/futurama.jpg',
              category: comedy)
```